### DISCUSSION TITLE
Do you guys know if you can add a deploy button to main branches for web apps? Would like to know if you could change the way the home page looks sorta to a decentralized App Store too.

### DISCUSSION THREAD

**Discussion by maiosx** [community/community#55670] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello I am writing wondering if there is a project for torrent handlers written like a new tab page that functions as a dapp downloader/ dapp store... and if you could build it? I think dapps are cool and so are web apps but don't want to run deployable env's all the time for them so do you guys plan on making a deploy engine/ interface in the future for browsers (based on p2p torrent handlers)?

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
